js/tests/Test_jsoo.ml

```diff
@@ -38,7 +38,7 @@ let skipped_tests =
     (* TODO: re-enable this when we fix the jsoo int overflow bug *)
     ("Go", [ 24 ]);
     (* TODO: investigate c_array_inits pattern parse error*)
-    ("C", [ 0 ]);
+    ("C", [ 2 ]);
```

Comment:
what are those index??? This seems very fragile. Why not simply put the name of the test file?
"C", ["c_array_inits"; ...]

Reply:
alcotest only lets you filter based on index, or the name (i.e. full rule > C NOT C tests/rules/c_array_inits.yaml..., so unless we want to include the file names in this test suite we're stuck w/ the index